ophyd/signal.py

```diff
@@ -839,8 +853,12 @@ def get(self, *, as_string=None, connection_timeout=1.0, form='time',
             as_string = self._string
 
         self.wait_for_connection(timeout=connection_timeout)
-        info = self._read_pv.get_with_metadata(as_string=as_string, form=form,
-                                               **kwargs)
+        self.log.info(
```

Comment:
There are two categories of log messages added in this PR:
1. Things that ophyd is doing
2. Things that ophyd is asking the control layer to do.

The log messages in category (2), such as this one, might better be done upstream in the control layers. (Caproto already goes it; pyepics could be made to do it.) _But_ since the immediate motivation for this additional logging is to debug a tricky pseudopositioners issue that may involve pyepics, including older versions of pyepics, I think we _should_ add these log messages, provisionally. In the longer term, we may submit a PR to pyepics and then strip these out after a couple of pyepics releases.

I propose issuing all these kinds of log messages from a separate logger named something like `'ophyd_control_layer'` not under the `ophyd.` logger namespace. That will serve two purposes: it will keep the noise down for users because the control layer-related messages will have to be separately turned up, and it will make it easier to specifically remove just these log messages in the future.